bin/create-release.sh

```diff
@@ -102,6 +107,23 @@ done
 # to older ones
 rm -rf ${ROOT}/${release_path}/dev_releases
 
+if find "${ROOT}/${release_path}/blobs/" -type l >/dev/null 2>/dev/null; then
```

Comment:
I don't understand why the presence of symlinks decides that we have to run `bosh reset-release`. What about the repos that don't have symlinks but still have v1 dev releases in the tree?

Reply:
If you have v1 dev releases, you have symlinks (since that's the output of BOSH CLI v1). If you don't, that's fine, we can just make the release with BOSH CLI v2.